plugin/service/storage/testing.go

```diff
@@ -147,9 +146,9 @@ func newTestMockS3(state *testMockS3State, opts ...testMockS3Option) s3APIFunc {
 			if cfg.Region != "" {
 				m.Region = cfg.Region
 			}
-			if cfg.EndpointResolverWithOptions != nil {
-				m.Endpoint = cfg.EndpointResolverWithOptions
-			}
+			// if cfg. != nil {
+			// 	m.Endpoint = cfg.EndpointResolverWithOptions
+			// }
```

Comment:
leftover comment from testing? should this be uncommented or removed?

Reply:
Good catch, I can't recall why because I wrote this a while back.

Reply:
I remember now, the aws-sdk-go-v2 use to configure endpoint resolvers with aws Config objects. That is no longer the case now, the endpoint resolvers are configured with api objects such as the s3 client or ec2 client. So this comment can be removed.